src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/WhatIsTheNewPrice.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using  SFA.DAS.EmployerCommitmentsV2.Web.RouteValues
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.WhatIsTheNewPriceViewModel
+
+@{
+    ViewBag.Title = "What is the agreed price of completing the training with  new training provider name? ";
```

Comment:
"new training provider name" needs replacing with the actual training provider name

Reply:
done